VGG16_Net.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>this might be a fair comparison against the previous method since the number of epochs are very different. however, from comparing results from the first 3 epochs, this does look to be better :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/Skin-Cancer-Classification/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>